index.js

```diff
@@ -12,12 +12,11 @@ module.exports = {
         node: true
     },
     parserOptions: {
-        ecmaVersion: 2017,
+        ecmaVersion: 2018,
         sourceType: 'module',
         ecmaFeatures: {
             generators: true,
-            objectLiteralDuplicateProperties: true,
-            experimentalObjectRestSpread: true
```

Comment:
This is

> 2. I updated all of the dependencies to the latest.

and following ESLint's migration guide updating from 4.* to 5.*: https://eslint.org/docs/user-guide/migrating-to-5.0.0#-the-experimentalobjectrestspread-option-has-been-deprecated